tasks/main.yml

```diff
@@ -50,3 +50,25 @@
   with_items: "{{ sonar_plugins }}"
   loop_control:
     loop_var: "sonar_cplugin"
+
+- meta: flush_handlers
+
+- name: "Ensure web-interface is started"
+  uri:
+    url: "http://{{ sonar_web_host }}:{{ sonar_web_port }}/{{ sonar_web_context }}/"
+    method: GET
+  register: _sonar_page
+  until: _sonar_page.status == 200
+  retries: 60
+  delay: 5
+  check_mode: no # because this module does not support check mode
+  ignore_errors: "{{ ansible_check_mode }}"
+
+- name: "Verify the Sonar version"
+  command: "grep 'SonarQube Server / {{ sonar_version }}' {{ sonar_logs_dir }}/web.log"
```

Comment:
is it better to check the api endpoint "api/server/version" ?
the documentation say it's unauthenticated IF "Force user authentication" option is OFF.

Reply:
Great idea. I'll try to check if this could be combined with a previous step where Ansible waiting for web UI.

Just checked with 7.9.5 - "api/server/version" return a version without attention to "Force user authentication" setting.  I even restarted the service to ensure that version "api/server/version" shown with "Force user authentication" ON. 

This behavior probably expected because Sonar web UI describe button with a notice "Some specific read-only Web APIs, including those required to prompt authentication, are still available anonymously."